libqtile/widget/battery.py

```diff
@@ -313,11 +314,15 @@ class Battery(base.ThreadedPollText):
         ('show_short_text', True, 'Show "Full" or "Empty" rather than formated text'),
         ('low_percentage', 0.10, "Indicates when to use the low_foreground color 0 < x < 1"),
         ('low_foreground', 'FF0000', 'Font color on low battery'),
-        ('update_delay', 60, 'Seconds between status updates'),
+        ('update_interval', 60, 'Seconds between status updates'),
         ('battery', 0, 'Which battery should be monitored'),
     ]
 
     def __init__(self, **config) -> None:
+        if "update_delay" in config:
+            warnings.warn("Change from using update_delay to update_interval for battery widget, removed in 0.15",
+                          DeprecationWarning)
```

Comment:
Should we patch it in to the right key here as you do below?

Reply:
Ah! yes totally missed that, thanks for the catch.